test/test_dependency_versions.py

```diff
@@ -21,12 +50,13 @@ def get_versions_from_constraint_file(constraint_file):
     return versions
 
 
-@pytest.mark.parametrize('python_version', ['2.7', '3.5', '3.8'])
-def test_pinned_versions(python_version):
+@pytest.mark.parametrize('python_version', ['2.7', '3.5', '3.6', '3.8'])
+def test_pinned_versions(tmp_path, python_version):
```

Comment:
Wondering if there's a (clean) way we wouldn't have to repeat `tmp_path` (and `tmp_path / 'project'`) each time. But I can't immediately make up something that's better.